src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizerConfigurationValidator.cs

```diff
@@ -26,19 +26,26 @@ public void Validate(AnonymizerConfiguration config)
                 // Grammar check on FHIR path
                 try
                 {
-                    compiler.Compile(rule[Constants.PathKey]);
+                    compiler.Compile(rule[Constants.PathKey].ToString());
                 }
                 catch (Exception ex)
                 {
                     throw new AnonymizerConfigurationErrorsException($"Invalid FHIR path {rule[Constants.PathKey]}", ex);
                 }
 
                 // Method validate
-                string method = rule[Constants.MethodKey];
+                string method = rule[Constants.MethodKey].ToString();
                 if (!Enum.TryParse<AnonymizerMethod>(method, true, out _))
                 {
                     throw new AnonymizerConfigurationErrorsException($"{method} not support.");
                 }
```

Comment:
if method looks like a number, exception will not be thrown. e.g. "8","100"

Reply:
Cool. This is a yuangu bug. I have fixed it now.

Reply:
AnonymizerMethod parsedMethod;

if (!Enum.TryParse(method, true, out parsedMethod) | !Enum.IsDefined(typeof(AnonymizerMethod), parsedMethod))
 {
        throw new AnonymizerConfigurationErrorsException($"Anonymization method {method} not supported.");
 }

Reply:
TryParse cannot handle valid numeric cases. I have updated to hashset version